lib/statesman/config.rb

```diff
@@ -12,5 +12,9 @@ def initialize(block = nil)
     def storage_adapter(adapter_class)
       @adapter_class = adapter_class
     end
+
+    def requires_new_transaction(requires_new)
```

Comment:
Might be clearer to call this something like `use_real_transactions`? Or invert it to `use_virtual_transactions`? The “requires new” terminology is very AR specific and probably doesn’t mean much to people who use other ORMs.

Reply:
True, I assumed since the option is only useful if you are using the AR adapter it would make sense.